src/shared/alb.ts

```diff
@@ -32,7 +32,7 @@ export default function alb(name: string, titlePrefix: string = ""): Component {
       type: "timeseries",
       requests: [
         {
-          q: `min:aws.elb.healthy_host_count{name:${name}}.rollup(min), max:aws.elb.un_healthy_host_count{name:${name}}.rollup(max)`,
```

Comment:
Is there a use case for this that I am missing? Are there times where it matters that there are multiple availability zones active and not just that there are enough tasks up?

Reply:
I'm looking at the old version and I don't really understand what it's trying to show. Does using the `min` mean it's showing the minimum in a particular AZ? Seems like what you've updated to would be more useful across the board!

Reply:
When I take a look at the `bastion` dashboard with the updated query, I seem to be getting values of ~300 for host count which seems a bit off though

Reply:
Removing the `.rollup(sum)` from both of the queries seems to produce better results though!

Reply:
That's weird, I am getting 5 as the host count with that query. I have removed rollup anyway.

The old version will give you the number of tasks in the AZ which currently has the least active tasks. The only use case I can see for it atm is that it would tell you if your tasks are not being represented in all of the AZs enough. For instance if you had 2 AZs and 0 tasks in 1 of them and 5 in the other you would see 0 in the current chart.